### PR TITLE
fixes #15454 - Changes online pulp backup to fully online backup

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

File: katello/katello-backup [hunk 1]
```diff
@@ -21,7 +21,7 @@ optparse = OptionParser.new do |opts|
       opts.abort("Previous backup directory does not exist: #{dir_path}")
     end
   end
-  
+
   opts.on("--online-backup", "Backup Pulp data online") do |online|
```
Comment (beav) [inline katello/katello-backup]: does this need to be altered to be something generic like "perform online backup"?

**Comment by beav** [comment]
I got the following error when I ran with `katello-restore /tmp/back` (no trailing slash):

```
pg_restore: [archiver] could not open input file "/tmp/backforeman.dump": No such file or directory
pg_restore: [archiver] could not open input file "/tmp/backcandlepin.dump": No such file or directory
```

**Comment by cfouant** [comment]
@beav - GOOD EYE on that! I've fixed it.

**Comment by beav** [comment]
tested and passed on el6

ACK, thanks!

**Comment by ehelms** [comment]
Thanks @cfouant !

**Pull request merged**